unity-ggjj/Assets/Scripts/BGScenes/BGSceneList.cs

```diff
@@ -26,13 +56,13 @@ private void Awake()
     /// <returns>The new active scene. Can be null if an error occurred.</returns>
     public BGScene SetScene(SceneAssetName sceneName) //TODO: Change this when making file names universal
```

Comment:
Nit: I believe this `TODO:` is safe to remove now. : D
```suggestion
    public BGScene SetScene(SceneAssetName sceneName)
```